src/models/KeyboardModel.ts

```diff
@@ -101,6 +102,13 @@ class KeymapItem {
     this.option = options[0];
     this.choice = options[1];
     this._toBeDelete = false;
+    // Check whether this key is an encoder or not
+    const positions = label.split('\n');
```

Comment:
I think you might integrate this code to line#100 which is the splitter of contents in a label.

Reply:
@adamrocker I refactored the logic, and I removed the `pos` instance variable because it is not used. Could you review them again? The diff is [8e714d7](https://github.com/remap-keys/remap/pull/734/commits/8e714d73fb63817e063c9ae882849dfeba3e271f).